docs/ota-client-guide/modules/ROOT/nav.adoc

```diff
@@ -85,6 +85,7 @@ ifndef::env-github[:pageroot:]
 * xref:{pageroot}build-only-ostree.adoc[Build only the OSTree part]
 * xref:{pageroot}rollback.adoc[Set up rollback behavior]
 * xref:{pageroot}deb-package-install.adoc[Install the client from a deb package]
+* xref:{pageroot}upload-large-binary.adoc[Upload a Large Binary file]
```

Comment:
Typically we follow sentence capitalization except for proper names, which I don't think applies to "Large Binary". :)

Reply:
@patrickvacek good point. I'll make the change.

Reply:
The reference here in the nav is still capitalized inconsistently.

Reply:
@patrickvacek it's strange that my changes weren't committed. I'll try it again.